config.json

```diff
@@ -1095,7 +1117,13 @@
         "name": "Resistor Color Trio",
         "uuid": "7170d6a3-a32f-44d7-b82e-524485c58aaf",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": [
+          "array-analysis",
+          "classes",
+          "errors",
+          "math",
```

Comment:
`arithmetic-operators` instead of `math` is enough for this exercise, you only need `**` to solve it